prompt.go

```diff
@@ -47,28 +86,35 @@ type Prompt struct {
 }
 
 // PromptTemplates allow a prompt to be customized following stdlib
-// text/template syntax. If any field is blank a default template is used.
+// text/template syntax. Custom state, colors and background color are available for use inside
+// the templates and are documented inside the Variable section of the docs.
```

Comment:
IMO this is a great place to define more details for the templates that can be used.

Reply:
I'll add some details from the text/template library!